test/falco_test.py

```diff
@@ -426,8 +427,13 @@ def test(self):
         self.possibly_copy_driver()
 
         if self.package != 'None':
-            # This sets falco_binary_path as a side-effect.
-            self.install_package()
+            #As per https://github.com/falcosecurity/falco/issues/982, skipping the docker_package test cases
+            #due to falcobuilder/falco-tester container, old kernel versions module build issue(gcc-6/5 dependecy) on ppc64le platform
+            if platform.machine() == 'ppc64le' and self.package.startswith("docker:"):
```

Comment:
I suspect this check will disable the testing for both DEB and RPM ((ref)[https://github.com/falcosecurity/falco/blob/3693b16c917d6920adffac03e80d38c1b64317ea/test/falco_tests_package.yaml]).

Maybe I'm remembering it wrong, but this definitely needs to be double-checked :)